templates/observation.html

```diff
@@ -30,6 +31,11 @@
   </li>
 {% endblock %}
 {% block content %}
+
+    <div id="loader-frame" class="loader-frame">
```

Comment:
Likewise - the rest of the file has 2-space indents, the new parts have 4-space ones.